services/ops/LogStatisticsAgent/logstatisticsagent/agent.py

```diff
@@ -87,6 +99,9 @@ def publish_analysis(self):
         Publishes file's size increment in previous time interval (60 minutes) with timestamp.
         Also publishes standard deviation of file's hourly size differences every 24 hour.
         """
+        if not self.configured:
```

Comment:
Remove this onstart block and just use the configure main section.  There should be no need for both blocks.